VSTextField.swift

```diff
@@ -138,13 +153,18 @@ class VSTextField: UITextField {
      Final text without formatting characters (read-only)
      */
     var finalStringWithoutFormatting : String {
-        return VSTextField.makeOnlyDigitsString(_textWithoutSecureBullets)
+        return makeOnlyDigitsString(_textWithoutSecureBullets)
     }
-    
-    
-    // MARK: - class methods
-    class func makeOnlyDigitsString(_ string: String) -> String {
-        let stringArray = string.components(separatedBy: CharacterSet.decimalDigits.inverted)
+
+    /**
+     Note: This was a class method for some reason
+           Changed it to allow instance variable isHexadecimal to be used
```

Comment:
I made it a class method to explicitly show its independence on the current state of the text field. In fact, this should probably be a private extension of the ```String``` type. Something like:

```let filteredString = currentTextForFormatting.keepOnlyDigits(isHexadecimal: true) ```

Would you mind updating it that way?